modules/dfpAdServerVideo.js

```diff
@@ -22,6 +22,9 @@ import {
   parseUrl
 } from '../src/utils.js';
 import {DEFAULT_DFP_PARAMS, DFP_ENDPOINT, gdprParams} from '../libraries/dfpUtils/dfpUtils.js';
+import { vastsLocalCache } from '../src/videoCache.js';
+import { vastXmlEditorFactory } from '../libraries/video/shared/vastXmlEditor.js';
```

Comment:
this now includes the video library with dfpAdServerVideo. It's better than in core, but still way more than you need. It's best to extract just the bit you need.